src/services/railgun/core/load-provider.ts

```diff
@@ -126,7 +105,7 @@ const loadProviderForNetwork = async (
     poseidonMerkleVerifierV3Contract,
     tokenVaultV3Contract,
     fallbackProvider,
-    pollingProvider,
+    undefined, // pollingProvider is being deprecated
```

Comment:
Type error here. I do not think it's a breaking change to change the type to `?` on the functions that take the now deprecated polling provider such that you no longer have to pass them